modules/init.txt

```diff
@@ -66,3 +66,5 @@ renamer
 #-----------------------
 #		Custom
 #----------------------
+custom/sql/mob_spawn_points.sql
+custom/sql/mob_zones
```

Comment:
I'm concerned about adding these lines in. I think anything in the costume folder should be left up for people to do locally or for their own server.

Reply:
Took these out.  Not sure why it picked up my init.txt when I have it on ignore.  The updated push will have this out.